equinox-shipment/Domain/Shipment.fs

```diff
@@ -0,0 +1,67 @@
+﻿module Shipment
+
+open System
+
+type Shipment = { id: string; containerId: string }
+
+module Events =
+
+    let [<Literal>] CategoryId = "Shipment"
+
+    let (|ForClientId|) (clientId: string) = FsCodec.StreamName.create CategoryId clientId
+
+    type Event =
+        | ShipmentCreated    of shipmentId  : string
+        | ShipmentAssigned   of containerId : string
+        | ShipmentUnassigned of containerId : string
+        | Snapshotted        of Shipment
```

Comment:
snapshotting is not imporant for this aggregate, esp in this form - caching will cover the efficiency

Reply:
Removed thanks!